src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/IllegalImportCheckTest.java

```diff
@@ -93,6 +93,20 @@ public void testIllegalClasses()
         verify(checkConfig, getNonCompilablePath("InputIllegalImportDefault.java"), expected);
     }
 
+    @Test
+    public void testIllegalClasses1()
```

Comment:
Why dont just extend previous test instead of creating a new one?
They seem to be similar

Reply:
In the previous case we actually are checking for the class name and in this case it is just the * so i guess both need to be check

Reply:
Can you please rename it to make it clearer what the difference is?

Reply:
Since it also a test for the Illegal-Classes , So isn't the name is fine?

Reply:
It can be "testIllegalClassesStarImport"

Reply:
Basically it is exceptional test case for the classes which kills the mutation.In this case keeping the regex value always true which is mutation doesn't work.

Reply:
> I think "testIllegalClassesStarImport" sounds better

This seems fine.I will edit it

Reply:
I understand, but it is better to make name that shows idea of test case.

Reply:
I got it ! Made the respective changes! Thanks for the help